tools/utils.hh

```diff
@@ -143,6 +143,7 @@ class tool_app_template {
 public:
     struct config {
         sstring name;
+        sstring short_name; // just the name, without the "scylla-" prefix
```

Comment:
I just wonder, why do we need the "long" name, the one *with* the "scylla-" prefix, at all?
Shouldn't we have just the name, say "nodetool" - and then if you really want - some logic that also allows "scylla-"+name? Without each tool needing to repeat this logic.

Reply:
Yes, the two names do look a bit redundant, especially that the only difference is a consistently common "scylla-" prefix. I will see if I can ditch this, making this simpler.

Reply:
Made `app_name` short in v2, dropped `app_name_short`.